sql/fxa_users_last_seen_v1.init.sql

```diff
@@ -5,9 +5,9 @@ SELECT
   IF(country IN ('United States', 'France', 'Germany', 'United Kingdom', 'Canada'),
     submission_date,
     NULL) AS date_last_seen_in_tier1_country,
-  * EXCEPT (submission_date, generated_time)
+  * EXCEPT (submission_date, generated_time, seen_in_tier1_country)
 FROM
   fxa_users_daily_v1
 WHERE
-  -- 2017-10-01 is the first date in the Amplitude FxA project.
-  submission_date = DATE '2017-10-01'
+  -- Choose an arbitrary date that contains no records.
+  submission_date = DATE_FROM_UNIX_DATE(0)
```

Comment:
❤️

Reply:
I ran this and verified that the table schema is created, but no partition is created, which is exactly what I had hoped for.